src/scripts/run-phase

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
There should be some (brief) prose added to the README about `run-phaseX` and how to use them.